doc/api/http2.md

```diff
@@ -108,13 +108,14 @@ have occasion to work with the `Http2Session` object directly, with most
 actions typically taken through interactions with either the `Http2Server` or
 `Http2Stream` objects.
 
+
+#### Http2Session and Sockets
```

Comment:
Is moving the heading intentional? It seems the next paragraph is out of topic now?

Reply:
Nope, I tried to resolve conflicts via browser and things didn't went well.

Reply:
I will squash my commits and update the PR.